exporters/prometheus/include/opentelemetry/exporters/prometheus/exporter.h

```diff
@@ -0,0 +1,107 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+#ifndef ENABLE_METRICS_PREVIEW
+#  include <memory>
+#  include <string>
+#  include <vector>
+
+#  include "opentelemetry/common/spin_lock_mutex.h"
+#  include "opentelemetry/exporters/prometheus/collector.h"
+#  include "opentelemetry/nostd/span.h"
+#  include "opentelemetry/sdk/metrics/data/metric_data.h"
+#  include "opentelemetry/sdk/metrics/instruments.h"
+#  include "opentelemetry/sdk/metrics/metric_exporter.h"
+#  include "opentelemetry/sdk/metrics/recordable.h"
+#  include "opentelemetry/version.h"
+#  include "prometheus/exposer.h"
+
+/**
+ * This class is an implementation of the MetricsExporter interface and
+ * exports Prometheus metrics data. Functions in this class should be
+ * called by the Controller in our data pipeline.
+ */
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+
+namespace exporter
+{
+namespace metrics
+{
+class PrometheusExporter : public sdk::metrics::MetricExporter
+{
+public:
+  /**
+   * Constructor - binds an exposer and collector to the exporter
+   * @param address: an address for an exposer that exposes
+   *  an HTTP endpoint for the exporter to connect to
+   */
+  PrometheusExporter(std::string &address);
```

Comment:
nit - To be consistent with other exporters, we can pass a struct with all configurable options (though here it would have only one option as address/endpoint). This `address` is the address of Collector process from which the Prometheus server does the pull?

Reply:
thanks, using option now.